### PR TITLE
react-homework-w1 -- Shyar

### PR DESCRIPTION
**Pull request by ShyarK**

### CODE REVIEW

File: homework-submission/todo-app/src/components/todoStatic.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+const TodoStatic = () => {
```
Comment (ArcoMul) [inline homework-submission/todo-app/src/components/todoStatic.jsx]: In the homework you are asked to use for the static list the same components as for the dynamic list. So to render the list item using your `<Todo>` component.

File: homework-submission/todo-app/src/components/todo.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './todo.css';
+const Todo = props => {
+  const { description, id, handleDone, done, deadline, onDelete } = props;
+  return (
+    <li className={done ? 'isDone' : ''}>
+      <span onClick={() => handleDone(id)}>
```
Comment (ArcoMul) [inline homework-submission/todo-app/src/components/todo.jsx]: Nice work! You want straight for the homework of week 2, implementing click events! :-)

File: homework-submission/todo-app/src/App.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+function App() {
+  const [todoListState, setTodoListState] = useState(todoList);
+
+  const handleDone = id => {
+    const todoIndex = todoListState.findIndex(todo => todo.id === id);
+    const todo = { ...todoListState[todoIndex] };
+    todo.done = !todo.done;
+    const todoList = [...todoListState];
+    todoList[todoIndex] = todo;
+    setTodoListState([...todoList]);
```
Comment (ArcoMul) [inline homework-submission/todo-app/src/App.js]: `todoList` is already a copy of your initial state, not need to make another copy before setting it as your new state

File: homework-submission/todo-app/src/App.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  };
+
+  const removeHandler = id => {
+    const todoList = [...todoListState];
+    const todoIndex = todoListState.findIndex(todo => todo.id === id);
+    todoList.splice(todoIndex, 1);
+    setTodoListState([...todoList]);
```
Comment (ArcoMul) [inline homework-submission/todo-app/src/App.js]: Same here, you are making twice a copy using the spread operator

File: homework-submission/todo-app/src/App.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { todoList } from './services/todoContents';
+import Todo from './components/todo';
+import TodoStatic from './components/todoStatic';
+
+function App() {
+  const [todoListState, setTodoListState] = useState(todoList);
```
Comment (ArcoMul) [inline homework-submission/todo-app/src/App.js]: Cool usage of hooks to have a separate state for your todos!

File: homework-submission/todo-app/src/components/todoStatic.jsx (hunk 1 above)
Comment (ShyarK) [reply homework-submission/todo-app/src/components/todoStatic.jsx]: @ArcoMul Thanks a lot to review my homework and give some feedback! Then I'll delete the todoStatic component and I'll put all the list (static & dynamic) in one component!

File: homework-submission/todo-app/src/App.js (hunk 3 above)
Comment (ShyarK) [reply homework-submission/todo-app/src/App.js]: OK, I'll change it, I think I don't even need to assign it to a variable, can I do it like this?
```
  todoListState[todoIndex] = todo;
  setTodoListState([...todoListState]);
```

**Pull request closed without merging**